### ISSUE TITLE
Adjust makemanpages.pl script to work with non-debian distros

### ISSUE THREAD

**Issue by binwiederhier** [syncany/syncany#148]
See https://github.com/syncany/syncany/blob/96dcc2af4e8b3bc65cce7aaf67ebe3a95692f7ab/gradle/debian/makemanpages.pl

**Comment by binwiederhier** [comment]
@pimotte 
Does the arch package come with man pages? No, right? Maybe we should do something about that :)

**Comment by pimotte** [comment]
I recall at some point having tried to make this script a bit more OS-independent, but I got a headache trying to do that and since --help gives the exact same text (right?), this hasn't been to high of a priority for me.

What would be needed for me to include the man pages in the arch packages, is that the result of this script is included in the tarball. If that's easier to do than I currently think it is, do feel free to give me some pointers/do it yourself:P

**Comment by binwiederhier** [comment]
https://github.com/syncany/syncany/commit/d78931871ceabdb64d7d17e88de349f7c87feaf3 makes it really easy for you. It decouples the "makeman.pl" script from Debian entirely:

```bash
$ ./gradle/man/makeman.pl 
Usage: makeman [INFILE] [OUTFILE]

Parameters:
  INFILE   Text file with structure of a man page (optional).
  OUTFILE  File at which to write the man page (optional).

  Both parameters are optional. If - is given, STDIN/STDOUT is used.

Examples:
  makeman sy.skel sy.1             # Creates man page at sy.1
  cat sy.skel | makeman - > sy.1   # Same result
  cat sy.skel | makeman - sy.1     # Same result
```

**Comment by pimotte** [comment]
8379372f4568ad0e82bbe72699ed4b46e2a7c3ff cannot possibly be how to properly include the manpages in the distributables, but I tried using inputs and outputs (see http://gradle.org/docs/current/userguide/application_plugin.html and http://gradle.org/docs/current/userguide/more_about_tasks.html#sec:task_inputs_outputs) and that complained about "outputs" not having a dir property. 

In the end this is how I managed to coerce gradle into doing this for me, but I don't believe this is reasonable. At all. Any pointers?

**Comment by binwiederhier** [comment]
From what I can see (without testing it), it looks good to me. The only thing we might want to consider is that it now doesn't compile anymore on Windows -- which is obviously a rather big deal. So either we simply don't include man pages when run on Windows, or we manually call the task when creating a dist.

**Comment by pimotte** [comment]
#388 is something I am happy with. It is now done is such a way I no longer have the feeling I'm strangling gradle. If you are good with this, we can merge it and I'll edit the PKGBUILD once 0.4.3 is here.

**Comment by binwiederhier** [comment]
Closed by #388
